support/hooks/pre-push

```diff
@@ -22,9 +22,9 @@ do
   else
     if [ "$REMOTE_SHA" = $DUMMY_SHA ]
     then
-      # New branch. Verify last 15 commits, since verification of the entire
+      # New branch. Verify the last commit, since verification of the entire
       # history would take too much time.
```

Comment:
I think we could just say "because that's where the changed code is" since there is a pretty large semantic difference between checking branch history and not.

Reply:
Yes this fixes that.

Reply:
Pedantically, it's not always where the changed code is. Someone could push 4 commits if they chose, but no one really does this. I can change the comment if you like.